config/config.go

```diff
@@ -281,6 +282,7 @@ func NewConfig() *Config {
 	cfg.RoutesURLs = commaListFlag()
 	cfg.ForwardedHeadersList = commaListFlag()
 	cfg.ForwardedHeadersExcludeCIDRList = commaListFlag()
+	cfg.CompressEncodings = commaListFlag("gzip", "br", "deflate")
```

Comment:
I think the default should be `"gzip", "deflate", "br"` to keep backward compatibility

Reply:
sure I can change, it's only to enforce the allowed options here